### ISSUE TITLE
CT: unable to find node from unique Trail id

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/scenery-phet#605]
scenery-phet has been failing CT since yesterday afternoon. Looks like changes to PDOMPointer made by @jessegreenberg for phetsims/scenery#1048

```
scenery-phet : interactive-description-fuzzBoard : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589932208678/scenery-phet/scenery-phet_en.html?continuousTest=%7B%22test%22%3A%5B%22scenery-phet%22%2C%22interactive-description-fuzzBoard%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1589932208678%22%2C%22timestamp%22%3A1589937430844%7D&brand=phet&ea&fuzzBoard&supportsDescriptions&memoryLimit=1000
Query: brand=phet&ea&fuzzBoard&supportsDescriptions&memoryLimit=1000
Uncaught Error: Assertion failed: unable to find node from unique Trail id
Error: Assertion failed: unable to find node from unique Trail id
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589932208678/assert/js/assert.js:22:13)
at Function.Trail.fromUniqueId (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589932208678/scenery/js/util/Trail.js:957:17)
at PDOMPointer.updateTrail (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589932208678/scenery/js/input/PDOMPointer.js:97:25)
at Input.dispatchA11yEvent (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589932208678/scenery/js/input/Input.js:1024:38)
at Input.focusinAction.Action.phetioPlayback (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589932208678/scenery/js/input/Input.js:560:14)
at Action.execute (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589932208678/axon/js/Action.js:225:18)
at HTMLDivElement.<anonymous> (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589932208678/scenery/js/input/Input.js:733:30)
at KeyboardFuzzer.chooseNextElement (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589932208678/scenery/js/accessibility/KeyboardFuzzer.js:72:21)
at KeyboardFuzzer.fuzzBoardEvents (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589932208678/scenery/js/accessibility/KeyboardFuzzer.js:161:10)
at https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589932208678/joist/js/Sim.js:263:27
id: Bayes Chrome
Snapshot from 5/19/2020, 5:50:08 PM
```

**Comment by pixelzoom** [comment]
Looks like this is also showing up in coulombs-law:

```
coulombs-law : interactive-description-fuzzBoard : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/coulombs-law/coulombs-law_en.html?continuousTest=%7B%22test%22%3A%5B%22coulombs-law%22%2C%22interactive-description-fuzzBoard%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1589958265229%22%2C%22timestamp%22%3A1589961041724%7D&brand=phet&ea&fuzzBoard&supportsDescriptions&memoryLimit=1000
Query: brand=phet&ea&fuzzBoard&supportsDescriptions&memoryLimit=1000
Uncaught Error: Assertion failed: unable to find node from unique Trail id
Error: Assertion failed: unable to find node from unique Trail id
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/assert/js/assert.js:22:13)
at Function.Trail.fromUniqueId (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/scenery/js/util/Trail.js:957:17)
at PDOMPointer.updateTrail (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/scenery/js/input/PDOMPointer.js:97:25)
at Input.dispatchA11yEvent (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/scenery/js/input/Input.js:1024:38)
at Input.focusinAction.Action.phetioPlayback (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/scenery/js/input/Input.js:560:14)
at Action.execute (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/axon/js/Action.js:225:18)
at HTMLDivElement.<anonymous> (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/scenery/js/input/Input.js:733:30)
at KeyboardFuzzer.chooseNextElement (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/scenery/js/accessibility/KeyboardFuzzer.js:72:21)
at KeyboardFuzzer.fuzzBoardEvents (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/scenery/js/accessibility/KeyboardFuzzer.js:161:10)
at https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/joist/js/Sim.js:263:27
id: Bayes Chrome
Snapshot from 5/20/2020, 1:04:25 AM
```

**Comment by pixelzoom** [comment]
And greenhouse-effect:

```
greenhouse-effect : interactive-description-fuzzBoard : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/greenhouse-effect/greenhouse-effect_en.html?continuousTest=%7B%22test%22%3A%5B%22greenhouse-effect%22%2C%22interactive-description-fuzzBoard%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1589958265229%22%2C%22timestamp%22%3A1589967857787%7D&brand=phet&ea&fuzzBoard&supportsDescriptions&memoryLimit=1000
Query: brand=phet&ea&fuzzBoard&supportsDescriptions&memoryLimit=1000
Uncaught Error: Assertion failed: unable to find node from unique Trail id
Error: Assertion failed: unable to find node from unique Trail id
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/assert/js/assert.js:22:13)
at Function.Trail.fromUniqueId (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/scenery/js/util/Trail.js:957:17)
at PDOMPointer.updateTrail (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/scenery/js/input/PDOMPointer.js:97:25)
at Input.dispatchA11yEvent (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/scenery/js/input/Input.js:1024:38)
at Input.focusinAction.Action.phetioPlayback (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/scenery/js/input/Input.js:560:14)
at Action.execute (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/axon/js/Action.js:225:18)
at HTMLDivElement.<anonymous> (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/scenery/js/input/Input.js:733:30)
at KeyboardFuzzer.chooseNextElement (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/scenery/js/accessibility/KeyboardFuzzer.js:72:21)
at KeyboardFuzzer.fuzzBoardEvents (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/scenery/js/accessibility/KeyboardFuzzer.js:161:10)
at https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/joist/js/Sim.js:263:27
id: Bayes Chrome
Snapshot from 5/20/2020, 1:04:25 AM
```

**Comment by pixelzoom** [comment]
And tambo:

```
greenhouse-effect : interactive-description-fuzzBoard : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/greenhouse-effect/greenhouse-effect_en.html?continuousTest=%7B%22test%22%3A%5B%22greenhouse-effect%22%2C%22interactive-description-fuzzBoard%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1589958265229%22%2C%22timestamp%22%3A1589967857787%7D&brand=phet&ea&fuzzBoard&supportsDescriptions&memoryLimit=1000
Query: brand=phet&ea&fuzzBoard&supportsDescriptions&memoryLimit=1000
Uncaught Error: Assertion failed: unable to find node from unique Trail id
Error: Assertion failed: unable to find node from unique Trail id
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/assert/js/assert.js:22:13)
at Function.Trail.fromUniqueId (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/scenery/js/util/Trail.js:957:17)
at PDOMPointer.updateTrail (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/scenery/js/input/PDOMPointer.js:97:25)
at Input.dispatchA11yEvent (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/scenery/js/input/Input.js:1024:38)
at Input.focusinAction.Action.phetioPlayback (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/scenery/js/input/Input.js:560:14)
at Action.execute (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/axon/js/Action.js:225:18)
at HTMLDivElement.<anonymous> (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/scenery/js/input/Input.js:733:30)
at KeyboardFuzzer.chooseNextElement (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/scenery/js/accessibility/KeyboardFuzzer.js:72:21)
at KeyboardFuzzer.fuzzBoardEvents (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/scenery/js/accessibility/KeyboardFuzzer.js:161:10)
at https://bayes.colorado.edu/continuous-testing/ct-snapshots/1589958265229/joist/js/Sim.js:263:27
id: Bayes Chrome
Snapshot from 5/20/2020, 1:04:25 AM
```

**Comment by pixelzoom** [comment]
This same failure is occurring in coulombs-law, greenhouse-effect, and tambo.

**Comment by jessegreenberg** [comment]
Thanks, the related issue is https://github.com/phetsims/scenery/issues/921. Offending lines were removed earlier today until this can be fixed.

**Comment by jessegreenberg** [comment]
This was fixed in https://github.com/phetsims/scenery/issues/921 and is no longer showing up on CT.
